atc/metric/emitter/prometheus.go

```diff
@@ -334,13 +340,62 @@ func (config *PrometheusConfig) NewEmitter() (metric.Emitter, error) {
 		prometheus.CounterOpts{
 			Namespace: "concourse",
 			Subsystem: "lidar",
-			Name:      "checks_total",
-			Help:      "Counts the number of checks finished",
+			Name:      "checks",
+			Help:      "Counts the number of checks performed",
 		},
-		[]string{"scopeID", "checkName"},
+		[]string{"scopeID", "checkName", "checkStatus"},
 	)
 	prometheus.MustRegister(checksVec)
 
+	checkEnqueueVec := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "concourse",
+			Subsystem: "lidar",
+			Name:      "check_enqueue",
+			Help:      "Counts the number of checks enqueued",
+		},
+		[]string{"checkName"},
+	)
+	prometheus.MustRegister(checkEnqueueVec)
+
+	checkQueueVec := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "concourse",
+			Subsystem: "lidar",
+			Name:      "check_queue_size",
+			Help:      "Records the size of check queue",
+		},
+		[]string{},
```

Comment:
if we're not passing any labels, it might be better to go with a regular Counter (not `Vec`)